test/studies/amr/lib/grid/Grid_def.chpl

```diff
@@ -40,8 +40,8 @@ class Grid {
 
   const dx: dimension*real;
           
+  const cells:          domain(dimension, stridable=true);
   const extended_cells: domain(dimension, stridable=true);
-  const cells:          subdomain(extended_cells);
```

Comment:
Local variables might also allow this to remain in the original order (the assignment to cells in the initializer could be an assignment to a local variable which is used to create extended_cells before it is copied into the cells field).  I think this was the right modification to make, though.